src/bot.js

```diff
@@ -12,21 +14,22 @@ const bot = new Discord.Client({
 });
 
 // Import commands
-const commands = require('./commands/index');
```

Comment:
`index.js` is required implicitly if the require path points to a folder.

These are equivalent:
```javascript
require('./commands')
require('./commands/index')
require('./commands/index.js')
```

Reply:
Nice to know fact, will apply to future (if any) projects.